File: Lookup.Dapper/Models/ApplicationUser.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Amics.Lookup.Models
{
    // reference to LstUsersList from ngc
    public class ApplicationUser
    {
        public string UserId { get; set; }
        public string UserName { get; set; }
        public string FirstName { get; set; }
        public string Password { get; set; }
        public string Signature { get; set; }
        public string Warehouse { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string UserDataBase { get; set; }
        public int Buyer { get; set; }
        public int SalesPerson { get; set; }
        public int WebAccess { get; set; }
        public int AmicsUser { get; set; }
        public int EmpList { get; set; }
        public int InvTrans { get; set; }
        public string ForgotPwdAns { get; set; }

    }
}

File: Amics.web/ClientApp/src/app/shared/models/application-user.ts
export class ApplicationUser {
  /**
   * Application User View Model.
   */
  constructor(
    public userId?: string,
    public userName?: string,
    public firstName?: string,
    public password?: string,
    public warehouse?: string,
    public lastName?: string,
    public email?: string,
    public userDataBase?: string,
    public buyer?: string,
    public salesPerson?: string,
    public webAccess?: string,
    public amicsUser?: string,
    public empList?: string,
    public invTrans?: string,
    public forgotPwdAns?: string
  ) {}

  isAuthenticated(): boolean {
    return !!this.userId;
  }
 
  isBasicUser(): boolean {
    return  true;
  }

  isAdmin(): boolean {
    return false;
  }

  isWebMaster(): boolean {
    return false;
  }
 
}
